cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -181,7 +181,8 @@
 
 {% macro render(controls, form, index) %}
     <div class="o-filterable-list-controls">
-        {% if request.GET %}
+        {% set has_active_filters = page.has_active_filters(request) %}
```

Comment:
yeah, I am going to stay with passing the `request` object as it gives you more flexibility if you decide to to different things in the method.